### DISCUSSION TITLE
Setting up CI job but unable to get past mcdev auth

### DISCUSSION THREAD

**Discussion by jnguyenandersen** [Accenture/sfmc-devtools#424] [Q&A]
I'm trying to set up a Azure DevOps CI job that runs a backup of SFMC.

I have a Powershell task that looks like this
`npm install -g mcdev`
`mcdev init --y.credentialsName "$(credentialName)" --y.client_id "$(clientId)" --y.client_secret "$(clientSecret)" --y.auth_url "$(authURL)" --y.gitRemoteUrl "$(gitRemoteURL)" --y.accountid $(accountId)`
`mcdev retrieve AW/BU_Name_Here`
`git commit -m "Backup" `
`git push -u origin`

There's a line of code here that's suppose to skip the interactive credentials prompt, but I'm still getting this error
16:11:20 error: .mcdev-auth.json not found. Please run 'mcdev init' to provide the missing credential details.

**Comment by JoernBerkefeld** [comment]
Hi @jnguyenandersen - if you want to execute multiple commands in a 1-liner you need to combine them with && as far as I know.
I'm guessing that's why things fail here.

Try running your 5 commands manually one at a time to confirm. Also, after running mcdev init with your parameters, post the log file here. It shouldn't contain sensitive data unless $(credentialName) or BU names include information you don't want to share. You would need to remove those from the log manually before sharing it here

**Comment by jnguyenandersen** [comment]
@JoernBerkefeld 
Sorry updated the format of my code. I actually have tried to separate each line out into it's own task. That was still the error message I received. I've attached a log and stripped out identifiable information:
[sfmcdevtoolserror.txt](https://github.com/Accenture/sfmc-devtools/files/9736160/sfmcdevtoolserror.txt)

**Reply by JoernBerkefeld** [reply]
Can you please try to run this on your own laptop? Mcdev creates its own log files in your projects log-folder and prints a more abstract version to cli. 

That log you sent is... horribly long and seems to contain a lot of other stuff 🤔
